eli5/catboost.py

```diff
@@ -0,0 +1,84 @@
+
+from __future__ import absolute_import,division
+
+import numpy as np
+import catboost
+
+from eli5.explain import explain_weights
+from eli5._feature_importances import get_feature_importance_explanation
+
+DESCRIPTION_CATBOOST = """
+CatBoost feature importances; values are numbers 0 <= x <= 1;
+all values sum to 1.
+"""
+
+
+@explain_weights.register(catboost.CatBoost)
+@explain_weights.register(catboost.CatBoostClassifier)
+@explain_weights.register(catboost.CatBoostRegressor)
+def explain_weights_catboost(catb,
+                            vec=None,
+                            top=20,
+                            importance_type='PredictionValuesChange',
+                            prettified=True,
```

Comment:
This argument looks unused, except for documentation (from the docs I didn't quite understand if we need it).

Reply:
In the lower function. `formatter` were using vec to filter. 
I will recheck.

Reply:
@lopuhin  In _feature_importances\get_feature_importance_explanation, vec is given as a positional argument and I cant make it a keyword argument since it is followed by a positional argument. Changing the order breaks xgboost and lightdbm implementation. So i have just added a `vec` variable as a quick fix in the latest commit.

Reply:
vec is not needed for catboost as it uses its internal vectorizer when given cat_indices parameter. 
Now my doubt here was should I write some more functions insplace of get_feature_importance_explanation() for catboost or just change the ordering of parameters in the files using this function.(eg xgboost,lightgbm)

Reply:
Sorry @AshwinB-hat I meant that ``prettified`` argument is unused - I should have mentioned it in the comment. Let me think about ``vec``...